docs/howto/operator/apache/cassandra.rst

```diff
@@ -0,0 +1,71 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+
+Apache Cassandra Operators
+======================
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite
+------------
+
+To use ``CassandraTableSensor`` and ``CassandraRecordSensor``, you must configure a :doc:`Cassandra Connection <../../connection/cassandra>`.
+
+:ref:`howto/operator:CassandraTableSensor`
+
+CassandraTableSensor
```

Comment:
Can you use titles that are more beginner-friendly?
Here is an example: https://airflow.readthedocs.io/en/latest/howto/operator/google/cloud/natural_language.html

Reply:
@mik-laj  Are these titles fine? - 
"Waiting for a Table to be created" for CassandraTableSensor
"Waiting for a Record to be created" for CassandraRecordSensor

Reply:
+1